docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -121,12 +121,23 @@ Going beyond that, you can specify an individual test method like this::
 Running the Selenium tests
 --------------------------
 
-Some tests require Selenium and a Web browser (Firefox, Google Chrome, or
-Internet Explorer). To allow those tests to be run rather than skipped, you must
-install the selenium_ package into your Python path and run the tests with the
-``--selenium`` option::
-
-   $ ./runtests.py --settings=test_sqlite --selenium admin_inlines
+Some admin tests require Selenium, a web browser and Python >= 2.6 to work via a
```

Comment:
The mentions of "Python >= 2.6" and selenium (version > 2.38) are from an old version of these docs and now obsolete. Make the modifications without reintroducing what was removed in a3473454ada67c0a16efeabcb78950641d4ac93c.